redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -138,7 +141,7 @@ DatabaseServerV4Response deleteByName(
     @ApiOperation(value = DatabaseServerOpDescription.DELETE_MULTIPLE_BY_CRN, notes = DatabaseServerNotes.DELETE_MULTIPLE_BY_CRN,
             consumes = MediaType.APPLICATION_JSON, nickname = "deleteMultipleDatabaseServersByCrn")
     DatabaseServerV4Responses deleteMultiple(
-            @ApiParam(DatabaseServerParamDescriptions.CRNS) Set<@ValidCrn String> crns,
+            @ApiParam(DatabaseServerParamDescriptions.CRNS) @ValidCrn(resource = DATABASE_SERVER) Set<String> crns,
```

Comment:
Question: does the Crn validation work with a collection of Crns as well?

Reply:
have you checked the change? anyway there is a new mock test for this to make sure

Reply:
sorry, so yes, it is also working with collection of crns